.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
+title: "[Bug/Performance]"
+labels:
+assignees: ''
+
+---
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**System information**
+- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
+- HW Platform (e.g. iPhone 8, Pixel 2, Samsung Galaxy, Intel based Mac, i7 based Dell with NVIDIA GPGPU etc., or Google Cloud machine)
+- TensorFlow installed from (source or binary):
+- TensorFlow version (use command below):
+- TensorFlow Addons version:
+- Python version and type(eg. Anaconda Python, Stock Python as in Mac, or homebrew installed Python etc):
+- Bazel version (if compiling from source):
+- GCC/Compiler version (if compiling from source):
+- CUDA/cuDNN version (if compiling/running with GPGPU):
+- GPGPU model and memory:
+
+You can collect some of this information using our environment capture [script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
+You can also obtain the TensorFlow version with
+python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"
```

Comment:
Same failure in TF2

Reply:
Why was this resolved? This script is failing for me in TF2 is it not for you?

Reply:
Yes it was failing for me too therefore i removed it for now

Reply:
Its still showing for me on this PR

Reply:
yes actually i git not add that commit adding it now

Reply:
@seanpmorgan The script is still working but tensorflow is not present in the dev environment should we modify this script according to our needs https://github.com/tensorflow/tensorflow/blob/master/tools/tf_env_collect.sh